paddle/fluid/eager/auto_code_generator/generator/eager_gen.py

```diff
@@ -60,6 +60,17 @@
 ]
 
 
+# white ops list whose kernel can be deleted after performance analysis
+# original kernel can be deleted when composite_grad kernel performs same to it
```

Comment:
original kernel and its derivative kernel

Reply:
done